java/lang/correctness/hardcoded-conditional.java

```diff
@@ -1,11 +1,13 @@
 class Bar {
-    void main() {
```

Comment:
did you intend to use this arg somewhere? or is it actually used and I just can't read java?

Reply:
yep, I see where! never mind